datalad/distributed/ora_export_archive.py

```diff
@@ -45,23 +45,23 @@
     exc_str,
 )
 
-lgr = logging.getLogger('datalad.customremotes.export_archive')
+lgr = logging.getLogger('datalad.customremotes.ora_export_archive')
 
 
 @build_doc
-class RIAExportArchive(Interface):
-    """Export an archive of a local annex object store for the RIA remote.
+class ORAExportArchive(Interface):
```

Comment:
I wonder, if we rename, should we rename to `export_archive_ora` or at least `export_ora_archive`?

Reply:
Agree. To me `export_archive_ora` feels better. We are exporting *for* use by ORA, not from something used by ORA.

Reply:
Done.